### PR TITLE
Indicate parameters on modern-root-site.md

### PR DESCRIPTION
**Pull request by esjimenezp**
Indicate parameters of the PowerShell to indicate which parameter used in the command

- [x] Update content

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/modern-root-site.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ If you've [turned on audit log search](/office365/securitycompliance/turn-audit-
 Invoke-SPOSiteSwap -SourceUrl https://contoso.sharepoint.com/sites/CommunicationSite -TargetUrl https://contoso.sharepoint.com -ArchiveUrl https://contoso.sharepoint.com/sites/Archive
 ```
 
-(Where SourceUrl is the site you want to use, TargetUrl is the root site you want to replace, and ArchiveUrl is the location where you want to archive the current root site.)
+(Where `-SourceUrl` is the site you want to use, `-TargetUrl` is the root site you want to replace, and `-ArchiveUrl` is the location where you want to archive the current root site.)
```
Comment (nenonix) [inline SharePoint/SharePointOnline/modern-root-site.md]:
```suggestion
(Where **-SourceUrl** is the site you want to use, **-TargetUrl** is the root site you want to replace, and **-ArchiveUrl** is the location where you want to archive the current root site.)
```

Comment (MikePlumleyMSFT) [reply SharePoint/SharePointOnline/modern-root-site.md]: Agreed

**Comment by scanum** [comment]
@esjimenezp Please correct changes suggested.

**Comment by scanum** [comment]
@esjimenezp Hi Eduardo, following up on this. Could you please make the changes that are being asked for? Thanks!

**Comment by scanum** [comment]
@esjimenezp Hi Eduardo, following up on this. Could you please make the changes that are being asked for? Thanks!

**Comment by scanum** [comment]
@MikePlumleyMSFT Hello! Wondering if this change is necessary. The submitter has not responded. I can have a freelancer look at this.

**Comment by MikePlumleyMSFT** [comment]
@kaarins?

**Pull request merged**